src/Analyser/NodeScopeResolver.php

```diff
@@ -1388,11 +1390,16 @@ private function processTraitUse(Node\Stmt\TraitUse $node, Scope $classScope, \C
 	 * @param string $traitName
 	 * @param \PHPStan\Analyser\Scope $classScope
 	 * @param \Closure $nodeCallback
+	 *
+	 * @return void
 	 */
 	private function processNodesForTraitUse($node, string $traitName, Scope $classScope, \Closure $nodeCallback)
 	{
 		if ($node instanceof Node) {
-			if ($node instanceof Node\Stmt\Trait_ && $traitName === (string) $node->namespacedName) {
+			if ($node instanceof Node\Stmt\Trait_
+				&& isset($node->namespacedName)
```

Comment:
AFAIK it's always set.

Reply:
But it’s not defined on `PhpParser\Node\Stmt\Trait` - is there somewhere where this property is documented/explicitly ignored in a config?